static/js/lightbox_panzoom.js

```diff
@@ -0,0 +1,67 @@
+import panzoom from "@panzoom/panzoom";
+
+import * as overlays from "./overlays";
+
+export class PanZoomControl {
+    constructor(container) {
+        this.container = container;
+        this.panzoom = panzoom(this.container, {
+            disablePan: true,
+            disableZoom: true,
+            cursor: "auto",
+        });
+
+        // keybindings for zoom control
+        // Original comment from lightbox_canvas.js:
```

Comment:
I'm wondering if this is worth looking in to addressing here? The dependency on overlays and adding an eventListener here on keydown feels bad as it's pretty hidden away.

Reply:
I think perhaps if we do end up removing the pan/zoom toggle, then we could move the hotkey part of this to the main `lightbox.js`, which already depends on overlays, and that'd feel more natural?

Reply:
I suppose another option would be to just have this block be a class defined in `lightbox.js`; I'm not sure we need an independent module for this class.

Reply:
> I suppose another option would be to just have this block be a class defined in `lightbox.js`; I'm not sure we need an independent module for this class.

That's a good point. I think I was cargo-culting the original lightbox_canvas module by putting this also in it's own module. There isn't really a good reason for it.